Parser/pegen/parse_string.c

```diff
@@ -931,6 +931,11 @@ fstring_find_expr(Parser *p, const char **str, const char *end, int raw, int rec
     /* Check for =, which puts the text value of the expression in
        expr_text. */
     if (**str == '=') {
+        if (p->feature_version < 8) {
+            RAISE_SYNTAX_ERROR("f-string: self documenting expressions are "
+                               "only supported in Python 3.8 and greater");
```

Comment:
Feel free to ignore this. (Note that if you don't the message should be changed in the old parser as well)
```suggestion
            RAISE_SYNTAX_ERROR("Self-documenting expressions in f-strings are "
                               "only supported in Python 3.8 and greater");
```

Reply:
All the other f-string SyntaxErrors start with "f-string: ", so I think it's good to stay consistent.

Reply:
Oh okay, didn't remember correctly then.